terraform/templates/cluster-test/ct

```diff
@@ -12,24 +12,27 @@ while (( "$#" )); do
     -c|--container)
       CONTAINER=$2
       shift 2
+      continue
```

Comment:
Why is this needed?

Reply:
When I passed multiple ct script specific flags to ct (eg `ct -c cluster-test-ci --image image-id`), it was only processing the first flag `-c cluster-test-ci` and ignoring the `--image image-id`. With this, the flags are being parsed correctly.

Reply:
This is very strange, can you give me a repro?
```
[ec2-user@cluster-test-runner ~]$ ct -c xxx --image image-id
Using image-id image
[ec2-user@cluster-test-runner ~]$ ct --image image-id -c xxx
Using image-id image
```

Reply:
I am unable to repro it, I must've made some mistake earlier. I have updated it to remove the continue statements